GitUI/BranchTreePanel/RepoObjectsTree.Designer.cs

```diff
@@ -388,7 +388,7 @@ private void InitializeComponent()
             this.mnubtnFetchRebase.Name = "mnubtnFetchRebase";
             this.mnubtnFetchRebase.Size = new System.Drawing.Size(266, 26);
             this.mnubtnFetchRebase.Text = "Fetch && Re&base";
-            this.mnubtnFetchRebase.ToolTipText = "Fetch & Rebase current branch to this branch";
+            this.mnubtnFetchRebase.ToolTipText = "Fetch & Rebase current branch on this branch";
```

Comment:
```suggestion
            this.mnubtnFetchRebase.ToolTipText = "Fetch then rebase current branch on this branch";
```

Reply:
Done with some other slight improvements on tooltips in this 'RepoObjectsTree.Designer.cs' file.